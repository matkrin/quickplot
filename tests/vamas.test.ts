import { assert, describe, expect, it } from 'vitest'
import fs from "fs";
import Vamas from "../src/vamas";

const file = fs.readFileSync("./tests/test_files/aes_staib.vms", "utf8");
const v = new Vamas(file);

describe('VAMAS Header', () => {
    const header = v.header;

    it('Institution Identifier', () => {
        assert.equal(header.institutionIdentifier, "Not Specified")
    })

    it('bar', () => {
        assert.equal(header.instrumentModelIdentifier, "Staib SuperCMA")
    })

    it('snapshot', () => {
        expect({ foo: 'bar' }).toMatchSnapshot()
    })
})
