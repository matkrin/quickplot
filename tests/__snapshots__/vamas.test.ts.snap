// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`VAMAS > snapshot 1`] = `
{
  "foo": "bar",
}
`;

exports[`VAMAS Header > snapshot 1`] = `
{
  "foo": "bar",
}
`;
